spec/form_objects/support/school/change_responsible_body_form_spec.rb

```diff
@@ -80,17 +80,17 @@
 
     context 'when the form is valid' do
       let(:school) { create(:school, :with_preorder_information) }
-      let(:new_responsible_body_id) { create(:trust).id }
+      let(:new_responsible_body) { create(:trust) }
 
-      subject(:form) { described_class.new(school: school, responsible_body_id: new_responsible_body_id) }
+      subject(:form) { described_class.new(school: school, responsible_body_id: new_responsible_body.id) }
```

Comment:
Sorry for the pedantry, but this could use: `responsible_body: new_responsible_body`